experimental/PartitionedPermutations/src/PartitionedPermutations.jl

```diff
@@ -0,0 +1,59 @@
+module PartitionedPermutations
+
+import Base: 
+    ==, 
+    *,
+    adjoint,
+    deepcopy,
+    deepcopy_internal,
+    hash,
+    size,
+    length
+
+import Oscar:
+    @req,
+    cycle_partition,
+    cycles,
+    is_dominated_by,
+    number_of_blocks,
+    Perm,
+    symmetric_group,
+    join,
+    compose,
+    lower_points,
+    upper_points,
+    SetPartition,
+    subsets,
+    Generic,
```

Comment:
```suggestion
```
The need to import `Generic` is basically always a sign that you use something that is not intended to be used directly. (like `Perm` in this case)